entity-service/src/test/kotlin/com/egm/stellio/entity/repository/Neo4jRepositoryTests.kt

```diff
@@ -1149,6 +1150,185 @@ class Neo4jRepositoryTests : WithNeo4jContainer {
         )
     }
 
+    @Test
+    fun `it should retrieve attribute list`() {
+        val firstEntity = createEntity(
+            "urn:ngsi-ld:Beehive:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Beehive"),
+            mutableListOf(
+                Property(name = "temperature", value = 36),
+                Property(name = "humidity", value = 65)
+            )
+        )
+        val secondEntity = createEntity(
+            "urn:ngsi-ld:Sensor:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Sensor"),
+            mutableListOf(
+                Property(name = "deviceParameter", value = 30),
+                Property(name = "isContainedIn", value = 61)
+            ),
+            NgsiLdGeoPropertyInstance.toWktFormat(GeoPropertyType.Point, listOf(24.30623, 60.07966))
+        )
+
+        createRelationship(EntitySubjectNode(firstEntity.id), "observedBy", secondEntity.id)
+
+        val attributeName = neo4jRepository.getAttribute()
+        val expectedValue = listOf(
+            "deviceParameter",
+            "https://uri.etsi.org/ngsi-ld/location",
+            "humidity",
+            "isContainedIn",
+            "observedBy",
+            "temperature"
+        )
+        assertEquals(6, attributeName.size)
+        assertArrayEquals(arrayOf(expectedValue), arrayOf(attributeName))
+    }
+
+    @Test
+    fun `it should retrieve details of attributes`() {
+        val firstEntity = createEntity(
+            "urn:ngsi-ld:Beehive:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Beehive"),
+            mutableListOf(
+                Property(name = "temperature", value = 36),
+                Property(name = "humidity", value = 65)
+            )
+        )
+        val secondEntity = createEntity(
+            "urn:ngsi-ld:Sensor:TESTC".toUri(),
+            listOf("https://ontology.eglobalmark.com/apic#Sensor"),
+            mutableListOf(
+                Property(name = "deviceParameter", value = 30),
+                Property(name = "isContainedIn", value = 61)
+            ),
+            NgsiLdGeoPropertyInstance.toWktFormat(GeoPropertyType.Point, listOf(24.30623, 60.07966))
+        )
+
+        createRelationship(EntitySubjectNode(firstEntity.id), "HAS_OBJECT", secondEntity.id)
+        createRelationship(EntitySubjectNode(secondEntity.id), "HAS_VALUE", firstEntity.id)
```

Comment:
I don't get what is the point of creating those 2 relationships using the core labels we use to identify properties and relationships

Reply:
removed the core labels from the test.